### ISSUE TITLE
How To Remove it or uninstall it 

### ISSUE THREAD

**Issue by fikrado** [adi1090x/rofi#114]
i have download half of the files and then run the installer script is their a way to remove it let me know

**Comment by Primordium** [comment]
It should be relative easy to uninstall everything. 
The setup script only copies some fonts to $HOME/.local/share/fonts
And the files inside the files folder to $HOME/.config/rofi.

So removing the files inside the rofi folder  should be enough.

**Comment by andro404-MC** [comment]
> It should be relative easy to uninstall everything. The setup script only copies some fonts to $HOME/.local/share/fonts And the files inside the files folder to $HOME/.config/rofi.
> 
> So removing the files inside the rofi folder should be enough.

True, also the installation script will back up your previous configs to ~/.config/rofi.$USER.

**Comment by virajshoor** [comment]
Not working
